THANKS.md

```diff
@@ -30,6 +30,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- markdownlint-enable -->
 <!-- prettier-ignore-end -->
+
```

Comment:
@iranzo can you please remove this redundant patch?

Reply:
I had to add it to avoid prettier failing the build on travis, I can remove it for sure

Reply:
But when trying to commit, local prettier will change it again

Reply:
I see. Thanks